configure.ac

```diff
@@ -32,15 +32,12 @@ AC_LANG(C)
 
 LT_INIT
 
-source $srcdir/versions.txt
-
 # Check for macros
 AC_DEFUN_ONCE([MACRO_CHECK],
 [
-	ifdef([$1], true, [AC_FATAL([Please install autoconf-archive version ${autoconf_archive_version}],)])
+	ifdef([$1], true, [AC_FATAL([Please install autoconf-archive],)])
 ])
 
-MACRO_CHECK([AX_VALGRIND_DFLT])
```

Comment:
Hi @jcvenegas - I agree that that would be a nice optimisation but I don't think that needs to block this PR (since we already fail if the expected version of `autoconf-archive` isn't installed). Could you raise an issue to capture the idea maybe?